### ISSUE TITLE
Problem: Module not found: Error: Cannot resolve module 'save-svg-as-png'

### ISSUE THREAD

**Issue by trevorbernard** [rrag/react-stockcharts#8]
I get the following error when using v0.2.0 of react-stockcharts. As a work around, I've added save-svg-as-png as a dependency in my project.

```
ERROR in ./~/react-stockcharts/lib/helper/SaveChartAsImage.js
Module not found: Error: Cannot resolve module 'save-svg-as-png' in react-stockcharts/lib/helper
 @ ./~/react-stockcharts/lib/helper/SaveChartAsImage.js 3:16-42
```

**Comment by rrag** [comment]
Thanks for the issue. I should not have included that in the `index.js`. the `save-svg-as-png` is not needed for the project to work, more of a test I was exploring to download the chart as an image.

I am currently in the middle of trying to make a change to improve the pan performance on firefox. Chrome is really fast, but firefox lags behind, even for canvas.

I will fix this issue along with the performance improvement change over the weekend.

**Comment by rrag** [comment]
published v0.2.1 with this fix
